src/toGamut.js

```diff
@@ -189,25 +189,18 @@ export function toGamutCSS (origin, { space = origin.space }) {
 		current.coords[1] = chroma;
 		if (min_inGamut && inGamut(current, space)) {
 			min = chroma;
-			continue;
 		}
 		else if (!inGamut(current, space)) {
 			const clipped = clip(current);
 			const E = deltaEOK(clipped, current);
+			// Note- this is missing several steps of the CSS Gamut Mapping
```

Comment:
That just sounds worrying without being informative

Reply:
I updated this in https://github.com/LeaVerou/color.js/pull/352/commits/b22d42be08af31944fad2e0ea8422217a35e941c to describe the steps I removed, and the reasoning.

Reply:
I'm pretty sure I also just use the equivalent of `E < JND`.

Reply:
Oh, never mind, I misunderstood. So this is my implementation. Ignore the LCh references as we subclass it for OkLCh updating the JND, space used, etc., but the logic is the same.  All of this code is tested and confirmed to be covered. I would be concerned about this over simplification of the algorithm.

https://github.com/facelessuser/coloraide/blob/main/coloraide/gamut/fit_lch_chroma.py

Reply:
Just in case there is any doubt that the deleted steps get exercised: https://app.codecov.io/gh/facelessuser/coloraide/blob/master/coloraide%2Fgamut%2Ffit_lch_chroma.py#L77.